examples/bare-metal/explorer_board/src/platform/app_pll_ctrl.h

```diff
@@ -4,7 +4,7 @@
 #ifndef APP_PLL_CTRL_H_
 #define APP_PLL_CTRL_H_
 
-#include "app_conf.h"
+#include "../app_conf.h"
```

Comment:
Could you return this to a non relative path, as it will prevent portability by forcing a specific file structure.